A42-xds-ring-hash-lb-policy.md

```diff
@@ -97,7 +97,10 @@ Here is how gRPC will handle each type of hash policy:
   (Note that we do not recommend that applications create multiple gRPC
   channels to the same virtual host, but if you do that, then the
   behavior here will not be exactly the same as using `connection_properties`,
-  because each channel may use a different endpoint.)
+  because each channel may use a different endpoint.). In order to facilitate
```

Comment:
> (Note that we do not recommend that applications create multiple gRPC
> channels to the same virtual host, but if you do that, then the
> behavior here will not be exactly the same as using `connection_properties`,
> because each channel may use a different endpoint.)

Did we want that behavior?  Because we could initialize this ID globally to a process if desired.


Also: nit: no period after the parenthetical sentence.  But should this sentence come before it, maybe?

Reply:
> Did we want that behavior? Because we could initialize this ID globally to a process if desired.

Changed wording to note that we do want even backend distribution across channels, which may or may not be in the same process or machine (i.e. we want even distribution across <i>all</i> channels).

We know of use cases that do use multiple channels within a process, and for those use cases, each channel should have affinity to its own backend (but hot spotting one backend is unnecessary for them).

> Also: nit: no period after the parenthetical sentence. But should this sentence come before it, maybe?

Moved things around to keep the parenthetical sentence at the end.